src/main/java/service/interfaces/FruitTypeService.java

```diff
@@ -0,0 +1,7 @@
+package service.interfaces;
+
+import model.Fruit;
+
+public interface FruitTypeService {
```

Comment:
You have GetOperationTypeService, why this one is not GetFruitTypeService

```suggestion
public interface GetFruitTypeService {
```